Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -36,6 +36,22 @@ file that was distributed with this source code.
         {% endif -%}
     </div>
 
+    <div id="field_actions_{{ id }}" class="field-actions">
+        {% set display_btn_add = sonata_admin.edit != 'admin' and sonata_admin.field_description.associationadmin.hasRoute('create') and sonata_admin.field_description.associationadmin.isGranted('CREATE') and btn_add %}
```

Comment:
please linebreak this

Reply:
Please line break anyway. If we follow this consistency rule, we can't make any change unless someone is willing to do it on all the files of all our libs.

Reply:
linebreaked ;)